R/fit_curves.R

```diff
@@ -156,6 +164,9 @@ fit_curves <- function(df_,
 #' @param n_point_cutoff integer indicating number of unique concentrations required to fit curve.
 #' @param capping_fold Integer value of the fold number to use for capping IC50/GR50. Default is \code{5}.
 #'
+#' @examples
+#' logisticFit(0.1, NA)
```

Comment:
I get the warning `In value[[3L]](cond) : not enough data points (0 < 4) to perform fitting`. Could you confirm or deny?

Reply:
I confirm. We use random data here just to show the usage